doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgis_sink.md

```diff
@@ -25,9 +30,86 @@ Colum should be provisioned as type [`geometry`](http://postgis.net/workshops/po
 
 [Top](#top)
 
+### <a name="section1.2"></a>Mapping `NGSIEvent`s to PostgreSQL data structures
+PostgreSQL organizes the data in schemas inside a database that contain tables of data rows. Such organization is exploited by `NGSIPostgreisSink` each time a `NGSIEvent` is going to be persisted.
```

Comment:
I'd suggest to add a note in this section, just after the title:

> **NOTE:** by the moment this mapping functionality works only for Postgis sink. However, the same will be implemented for the Postgresql sink itself. In that moment probably this documentation will be moved to the Postgresql sink documentation (as that is the common place for documentation which applies to both sinks)

Reply:
Fixed in [6507901](https://github.com/telefonicaid/fiware-cygnus/pull/1874/commits/650790136ad3675975c35cb6410d611230e4e2bd)